doc/user/content/release-notes.md

```diff
@@ -114,6 +114,9 @@ The presence of this comment ensures that PRs that are alive across a release
 boundary don't silently merge their release notes into the wrong place.
 {{</ comment >}}
 
+- Support the inverse [trigonometric functions](/sql/functions/#trigonometric-func)
+  `asin`, `asinh`, `acos`, `acosh`, `atan`, `atanh`.
+
```

Comment:
Can you toss this up above the "put release notes here" line?

Reply:
oops. fixed!